packages/web/src/components/cells/code.tsx

```diff
@@ -643,6 +680,27 @@ function CodeEditor({
 
   const updateCellOnServerDebounced = useDebouncedCallback(updateCellOnServer, DEBOUNCE_DELAY);
 
+  const formatCellCode = async (source: string, viewUpdate: ViewUpdate) => {
```

Comment:
Nit: There's no `await` inside this function so we don't need the `async` keyword in this case. Alternatively can change to using await and ditch the promise callbacks.